src/en/curriculum/index.html

```diff
@@ -2,45 +2,47 @@
 title: Accessibility Curriculum
 layout: layouts/base.njk
 description: The Accessibility Curriculum is a list of courses that will provide knowledge of accessibility to better serve internal and external clients.
+internalLinks: true
 ---
-		<section>
-			<h2>Intention and vision for the accessibility curriculum</h2>
-			<p>The new <a href="https://www.parl.ca/DocumentViewer/en/42-1/bill/C-81/royal-assent"><em>Accessible Canada Act</em></a> will require Innovation, Information and Technology Branch (IITB) employees to be aware of their obligations to ensure an accessible work environment. The list of courses will provide an opportunity for employees to increase their knowledge of accessibility and to better serve internal and external clients. It will also be an opportunity for various target audiences such as Project Managers, IITB developers, programmers and technicians to proactively provide accessible services, software and programs that bring the greatest value to our colleagues who use ACT (adaptive computer technology).</p>
-			<section>
-				<h3>IITB's mandate:</h3>
-				<ul>
-					<li>To provide modern, secure, effective and efficient information and technology services that bring the greatest value to clients</li>
-					<li>To securely connect information and technology for clients, partnerships and citizens.</li>
-				</ul>
-			</section>
-			<section>
-				<h3>IT Accessibility's mandate:</h3>
-				<ul>
-					<li>To be a leader in accessibility, promoting and enabling an all-inclusive workplace and Service to ESDC employees by providing adaptive technology and accessible solutions.</li>
-				</ul>
-			</section>
-		<h3>Target learners</h3>
+
+
+<section>
```

Comment:
Missing a marker or something to visually ensure the content at line 38+ are outside this level 2 section content